build/charts/antrea/values.yaml

```diff
@@ -207,6 +207,8 @@ agent:
 controller:
   # -- Port for the antrea-controller APIServer to serve on.
   apiPort: 10349
+  # -- Exporting antrea-controller APIServer via nodeport.
+  nodeport: 0
```

Comment:
this should be `nodePort` for consistency with other params and the rest of the code base. I actually think we should use `apiNodePort` to make it more consistent with `apiPort`.

Reply:
Updated it.